src/components/redemption/Redeeming.js

```diff
@@ -11,36 +11,39 @@ function Redeeming(props) {
   return <RedeemingComponent {...props} address={props.address || params.address} />
 }
 
-class RedeemingComponent extends Component {
-  componentDidMount() {
-    const { requestRedemption } = this.props
-
+const RedeemingComponent = ({ requestRedemption, error }) => {
+  useEffect(() => {
     requestRedemption()
-  }
+  }, [requestRedemption])
 
-  render() {
-    return (
-      <div className="confirming">
-        <div className="page-top">
-          <StatusIndicator pulse />
+  return (
+    <div className="confirming">
+      <div className="page-top">
+        <StatusIndicator pulse />
+      </div>
+      <div className="page-body">
+        <div className="step">
+          Step 2/6
+        </div>
+        <div className="title">
+          { error ? 'Error redeeming bond' : 'Redeeming...'}
         </div>
-        <div className="page-body">
-          <div className="step">
-            Step 2/6
-          </div>
-          <div className="title">
-            Redeeming...
-          </div>
-          <hr />
-          <div className="description">
-            <p>We’re waiting for you to confirm invoice details in your Wallet.</p>
-          </div>
+        <hr />
+        <div className="description">
+          <p>We’re waiting for you to confirm invoice details in your Wallet.</p>
+        </div>
+        <div className="error">
+          { error }
```

Comment:
Another (mostly leaving these so we have a complete compendium).

At this point there's probably some sort of component to be had.

Reply:
Added one in 766004b